### PR TITLE
tidb-config-file.md: add description for txn-local-latches

### PR DESCRIPTION
**Pull request by AndreMouche**
@coocood @tiancaiamao PTAL

### CODE REVIEW

**Comment by tiancaiamao** [comment]
LGTM

File: op-guide/tidb-config-file.md [hunk 1]
```diff
@@ -220,4 +220,16 @@ prepare 语句的 Plan cache 设置。
...
+## txn-local-latches
+
+事务内存锁相关配置，当本地事务冲突比较多时建议开启。
+
+### `enable`
++ 开启
++ 默认: true
+
+### `capacity`
++ Hash 对应的 slot 数。
```
Comment (coocood) [inline op-guide/tidb-config-file.md]: 需要说明一下，设置过大会占用更多内存，过小会导致延迟增大，性能下降

**Comment by tiancaiamao** [comment]
LGTM @coocood

**Comment by coocood** [comment]
LGTM

**Comment by coocood** [comment]
@lilin90 PTAL

**Comment by lilin90** [comment]
@coocood Got it. Thanks! I'll update it in docs later.

**Pull request merged**